packages/oui-criteria-adder/src/index.spec.js

```diff
@@ -160,6 +160,119 @@ describe("ouiCriteriaAdder", () => {
             });
         });
 
+        describe("Column type = boolean", () => {
+            let configuration;
+
+            beforeEach(inject(_ouiCriteriaAdderConfiguration_ => {
+                configuration = _ouiCriteriaAdderConfiguration_;
+            }));
+
+            it("should call function of onSubmit attribute, when form is submitted, with the model value", () => {
+                const propertyMeta = mockData.properties[2];
+                const value = {
+                    name: configuration.translations.true_label,
+                    value: true
+                };
+
+                // Initial condition
+                expect(propertyMeta.type).toEqual("boolean");
```

Comment:
(optional) it could be more declarative and simpler to understand without using an external file e.g.: controller.columnModel = generatePropertyMeta("boolean");
Ideally, expect should be only in the last part of the test, to divide a test in 3 distincts blocks (arrange-act-assert)